Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Oops, can you keep Gemfile and Gemfile.lock changes out of the PR unless they're feature-specific and required? This looks like versions changes only, right? Thanks!